# Disallow vino-server to register into the session since it is auto-started.
# Gentoo: https://bugs.gentoo.org/show_bug.cgi?id=277989
# Gnome: https://bugzilla.gnome.org/show_bug.cgi?id=579355
--- a/server/vino-server.desktop.in.in	2009-03-23 22:17:35.647373307 +0100
+++ b/server/vino-server.desktop.in.in	2009-03-23 22:17:53.079875343 +0100
@@ -1,7 +1,7 @@
 [Desktop Entry]
 _Name=Remote Desktop
 _Comment=GNOME Remote Desktop Server
-Exec=@LIBEXECDIR@/vino-server
+Exec=@LIBEXECDIR@/vino-server --sm-disable
 Icon=preferences-desktop-remote-desktop
 OnlyShowIn=GNOME;
 Terminal=false
